stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/DecoratorsProvider.kt

```diff
@@ -0,0 +1,5 @@
+package io.getstream.chat.android.ui.messages.adapter.viewholder.decorator
+
+public interface DecoratorsProvider {
+    public fun getDecorators(): List<Decorator>
```

Comment:
This could probably be just a `val`, looks like we're just returning a fixed, already existing list everywhere anyway